.github/workflows/test-r-scripts.yaml

```diff
@@ -63,13 +76,27 @@ jobs:
           r-version: ${{ matrix.config.r }}
           http-user-agent: ${{ matrix.config.http-user-agent }}
 
+      - name: Install system dependency of usethis >= 2.0.0
+        if: runner.os == 'Linux'
+        run: |
+          sudo apt install libgit2-dev
+
       - name: Install dependencies from CRAN
         run: |
           install.packages("rmarkdown")
           install.packages("knitr")
           install.packages("renv")
-          dependencies <- renv::dependencies()$Package
-          install.packages(dependencies)
+          deps <- unique(renv::dependencies("../PACTA_analysis")$Package)
+          exclude <- c(
+            "R",
+            "utils",
+            "base",
+            "tools",
+            "create_interactive_report",
+            "PACTA.analysis",
+            "StressTestingModelDev"
+          )
```

Comment:
renv seems to detect some false positives

Reply:
StressTestingModelDev is now a package, but create_interactive_report is not -- so I have no idea. I found this list in a warning on CI and copy-pasted without much experimentation -- so don't take my word (but don't worry because this here seems to do no harm). If I hadn't spent a whole day on this PR I would have tried to dig deeper. If you find a hint let me know.